packages/sysutils/ethmactool/scripts/ethmactool-config

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+MAC="000000000000"
+COMPATIBLE=$(/usr/bin/dtsoc)
+MAC_STEP=""
+
+validate_mac() {
+  [ ${#MAC} = 12 -a "${MAC}" != "000000000000" ]
```

Comment:
```suggestion
  [ ${#MAC} -eq 12 -a "${MAC}" != "000000000000" ]
```
as `${#MAC}` is a numeric result and shell should use `-eq` for numeric comparisons.